base/settings/R/site_pft_link_settings.R

```diff
@@ -34,7 +34,7 @@ site.pft.link.settings <- function(settings) {
        unique()
      
      new.pfts <-  pft.l %>%
-       purrr::drop(~.x %in% def.pfts) %>%
+       purrr::discard(~.x %in% def.pfts) %>%
```

Comment:
@bailsofhay can you make this change in your version of this file in pecan then re-install pecan settings then trying running the LAI SDA where you actually run the prepare settings step and see if that fixes your current bug?

Reply:
This change fixed our issue with purrr in prepare.settings